content/docs/uncontrolled-components.md

```diff
@@ -39,46 +39,80 @@ class NameForm extends React.Component {
 
 [**Try it on CodePen**](https://codepen.io/gaearon/pen/WooRWa?editors=0010)
 
-Since an uncontrolled component keeps the source of truth in the DOM, it is sometimes easier to integrate React and non-React code when using uncontrolled components. It can also be slightly less code if you want to be quick and dirty. Otherwise, you should usually use controlled components.
+Ponieważ komponent niekontrolowany zachowuje źródło prawdy w DOM, czasami łatwiej jest zintegrować kod reactowy z kodem niereactowym, gdy używa się komponentów niekontrolowanych. Dzięki temu może pojawić się również nieco mniej kodu, jeśli chcesz podejść do tematu z grubsza. W przeciwnym razie zwykle powinieneś używać komponentów kontrolowanych.
```

Comment:
```suggestion
Ponieważ komponent niekontrolowany zachowuje źródło prawdy w DOM, czasami łatwiej jest zintegrować kod reactowy z kodem niereactowym, gdy używa się komponentów niekontrolowanych. Wpłynie to również na mniejszą ilość kodu i pozwoli na wprowadzenie zmian "na szybko". W przeciwnym razie zwykle powinno się używać komponentów kontrolowanych.
```
1. Zdanie pośrodku brzmi wg mnie dziwnie. Moje też nie najlepiej, szczerze powiedziawszy, ale chyba udało mi się wyznaczyć kierunek ;) Zastanów się, jak to lepiej napisać.
2. W dokumentacji piszemy bezosobowo.

Reply:
1. Rozumiem, to trochę pomęczymy to zdanie wspólnie 
```suggestion
Wpłynie to również na mniejszą ilość kodu **i pozwoli na wprowadzenie zmian "na szybko"**
```
W oryginalnej wersji jest "It can also be slightly less code **if you want to be quick and dirty**"

Męczę to zdanie i męczę i wydaje mi się, że trzeba je całkiem przekształcić. W oryginale czuć, że to zdanie jest nacechowane negatywnie przez zwrot "quick and dirty" - myślę, że w naszej wersji możemy pominąć ten pejoratywny ładunek. Jeśli mamy używać bezokoliczników, to utrzymanie tego negatywnego ładunku jest zadaniem ponad moje możliwości :-P

Co do Twojej wersji (i mojej poprzedniej) określmy wyraźniej podmiot, zamiast używania zaimka "to", bo ciężko było mi załapać treść zdania. Trudno też było mi zrozumieć, dlaczego użycie komponentów niekontrolowanych, pozwoli na wprowadzenie zmian "na szybko".

Moja dalsza propozycja:

Użycie komponentów niekontrolowanych pozwala zaimplementować formularz "na szybko" dzięki mniejszej ilości kodu.

Reply:
1. Rozumiem, to trochę pomęczymy to zdanie wspólnie 
```suggestion
Wpłynie to również na mniejszą ilość kodu **i pozwoli na wprowadzenie zmian "na szybko"**
```
W oryginalnej wersji jest "It can also be slightly less code **if you want to be quick and dirty**"

Męczę to zdanie i męczę i wydaje mi się, że trzeba je całkiem przekształcić. W oryginale czuć, że to zdanie jest nacechowane negatywnie przez zwrot "quick and dirty" - myślę, że w naszej wersji możemy pominąć ten pejoratywny ładunek. Jeśli mamy używać bezokoliczników, to utrzymanie tego negatywnego ładunku jest zadaniem ponad moje możliwości :-P

Co do Twojej wersji (i mojej poprzedniej) określmy wyraźniej podmiot, zamiast używania zaimka "to", bo ciężko było mi załapać treść zdania. Trudno też było mi zrozumieć, dlaczego użycie komponentów niekontrolowanych, pozwoli na wprowadzenie zmian "na szybko".

Dodatkowo ostatnie zdanie "W przeciwnym razie..." traci sens, bo nie odwołujemy się już do "if you want to be quick and dirty".

Poniżej moja dalsza propozycja:

```suggestion
Ponieważ komponent niekontrolowany zachowuje źródło prawdy w DOM, czasami łatwiej jest zintegrować kod reactowy z kodem niereactowym, gdy używa się komponentów niekontrolowanych. Użycie komponentów niekontrolowanych pozwala zaimplementować formularz "na szybko" dzięki mniejszej ilości kodu. Zwykle jednak powinno używać się komponentów kontrolowanych.
```

Reply:
Druga wersja mi pasuje. Wrzucaj! :)